opm/simulators/flow/equil/InitStateEquil_impl.hpp

```diff
@@ -578,9 +578,11 @@ deriveSaturations(const Position& x,
     this->setEvaluationPoint(x, reg, ptable);
     this->initializePhaseQuantities();
 
-    if (ptable.waterActive()) { this->deriveWaterSat(); }
     if (ptable.gasActive())   { this->deriveGasSat();   }
 
+    if (ptable.waterActive()) { this->deriveWaterSat(); }
```

Comment:
Why is the order changed for the gasActive() and waterActive() related lines?

Nitpick: double blank line below.

Reply:
For 2p gas-water Sw = 1 - Sg. So we need to compute the gas saturation first.